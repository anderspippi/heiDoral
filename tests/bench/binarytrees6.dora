//= file bench/binarytrees/binarytrees2.dora
//= args 6
//= vm-args --gc=zero
//= stdout file