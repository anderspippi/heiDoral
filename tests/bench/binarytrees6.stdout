stretch tree of depth 7	 check: 255
64	 trees of depth 4	 check: 1984
16	 trees of depth 6	 check: 2032
long lived tree of depth 6	 check: 127
